boards/nrf51dk/README.md

```diff
@@ -8,30 +8,27 @@ radio. The kit is Arduino shield compatible and includes several
 buttons.  All code for the kit is compatible with the nRF51822 as
 well.
 
-## Necessary tools
+## Getting Started
 
-There are two ways to program the nRF51DK: JTAG or the mbed file
-system. If you choose to use JTAG (the recommended approach), this
-requires `JLinkExe`, a JTAG programming application.  It can be
-downloaded from [Segger](https://www.segger.com/downloads/jlink), you
-want the "Software and Documentation Pack".
+First, follow the [Tock Getting Started guide](../../doc/Getting_Started.md)
 
-## Programming the kernel
+JTAG is the preferred method to program. The development kit has an
+integrated JTAG debugger, you simply need to [install JTAG
+software](../../doc/Getting_Started.md#optional-requirements).
 
-### Programming with JTAG (recommended)
+### Programming the kernel
 
-The nRF51DK, a Segger JTAG chip is included on the board. Connecting
-the board to your computer over USB allows you to program (and debug)
-Tock with JTAG. To compile and install the Tock kernel on the nrf51dk
-using JTAG, follow the standard Tock instructions (the "Getting
-Started" guide).
+Once you have all software installed, you should be able to simply run
+`make flash` from this directory to install a fresh kernel.
 
-## Programming user-level applications
+### Programming user-level applications
 
-To compile and install compile applications for the nrf51dk, follow the
-standard Tock instructions (the "Getting Started" guide).
+After building (`make` in the application folder), you can program an
+application using tockloader:
 
-## Debugging
+    tockloader install --jtag --board nrf51dk --arch cortex-m0 build/blink.tab
```

Comment:
*build/blink.tab* indicates that the name of the app is blink right? 
Perhaps ```NAME_OF_THE_APP.tab``` is better?

I haven't used tockloader for this to be honest, you are sure this works?
Tried it briefly and it didn't work perhaps it was some problem with my installation tockloader